README.md

```diff
@@ -63,19 +74,19 @@ We don't know, since it depends on many factors. We will announce each new stabl
 
 Read [Contributing.md](.github/Contributing.md) and join our Discord to get started.
 
-### Is the A32NX payware?
+### Are the aircraft payware?
 
-No, it is a completely free aircraft, open-source.
+No, they are completely free aircraft, open-source.

```

Comment:
```suggestion
No, they are completely free aircraft, open-source, including SimBridge and other FlyByWire projects which are publicly accessible via GitHub.
```

This way we do not block any future developments/ideas.

Reply:
Might as well fix the weird phrasing too:
```
No, they are completely free and open source, along with SimBridge and other FlyByWire projects that are publicly accessible via GitHub.
```